### PR TITLE
docs: add more info about running a CN node on public networks

### LINKED ISSUE #2065: Extended CN documentation
We need some guide for running neo-go as CN on regular networks, `consensus.md` can be extended for that (referring to `cli.md` for various tuning options).

### PR DESCRIPTION
**Pull request by roman-khimov**
Fix #2065.

### CODE REVIEW

File: docs/consensus.md [hunk 1]
```diff
@@ -1,9 +1,125 @@
...
+```
+$ neo-go wallet candidate register -a NiKEkwz6i9q6gqfCizztDoHQh9r9BtdCNa -w wallet.json -r http://localhost:10332
+```
+where `NiKEkwz6i9q6gqfCizztDoHQh9r9BtdCNa` is your address, `wallet.json` is a
+path to NEP-6 wallet file and `http://localhost:10332` is an RPC node to
+use.
+
+This command will create and send appropriate transaction to the network and
+you should then wait for it to settle in a block. If all goes well it'll end
+with "HALT" state and your registration will be completed.
```
Comment (fyrchik) [inline docs/consensus.md:94]: Do we have any docs on how to wait for this? #2067 may be related.

**Review by fyrchik — APPROVED** [review]

File: docs/consensus.md (hunk 1 above)
Comment (AnnaShaleva) [reply docs/consensus.md:94]: Or we may just refer to the example of `getapplicationlog` RPC request for this transaction.

File: docs/consensus.md [hunk 2]
```diff
@@ -1,9 +1,125 @@
...
+
+### Voting
+
+After registration completion if you own some NEO you can also vote for your
+candidate to help it become CN and receive additional voter GAS. To do that
+you need to know the public key of your candidate, if you the wallet it can be
```
Comment (AnnaShaleva) [inline docs/consensus.md:100]: Is this a typo? `if you the wallet it...`

Comment (roman-khimov) [reply docs/consensus.md:100]: It is. Fixed.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**